fabric/src/main/java/org/mtr/mod/Init.java

```diff
@@ -184,14 +184,8 @@ public static void init() {
 					);
 				} else {
 					Main.LOGGER.error("Transport Simulation Core not responding; stopping Minecraft server!");
-					if (main != null) {
-						main.stop();
```

Comment:
Why are these `stop()` statements being removed?

Reply:
MTR have event hooks when a Minecraft server is stopping (REGISTRY.eventRegistry.registerServerStopping), which also stops TSC